.github/workflows/tests.yml

```diff
@@ -64,15 +64,19 @@ jobs:
                   key: build-output-${{ github.run_id }}
             - name: Build all
               run: npm run build && npm run build.debug
+            - name: Fetch fonts
+              run: npm run fetch-fonts
```

Comment:
This is needed to get the extensions screenshots to pass (otherwise they have a fallback font)